css/main.css

```diff
@@ -0,0 +1,664 @@
+  color: #f55800;
+  background: #fff5e1;
+  border: 1px solid #f55800;
+}
+
+.centered-link {
+  align-self: center;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+#contact {
+  padding: 10%;
+}
+
+#contact h2 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input {
+  border: none;
+  outline: none;
+  border-bottom: solid 2px #dbd8d7;
+}
+
+textarea {
+  color: #3c3a39;
+  border: none;
+  outline: none;
+}
+
+input,
+textarea {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 5px;
+  padding: 5px;
+  min-width: 50vw;
+}
+
+.name-form {
+  display: none;
+}
+
+.footer-social {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 5vh;
+  filter:
+    invert(67%) sepia(46%) saturate(986%) hue-rotate(179deg)
+    brightness(92%) contrast(82%);
+}
+
+/* styling of my modal ------------------------------------------ */
+.nav_modal-background {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  z-index: 6;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  backdrop-filter: blur(4px);
+  visibility: hidden;
+  background-color: transparent;
+}
+
+.nav_modal {
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  background-color: rgb(58, 58, 58);
+  z-index: 3;
+  flex-direction: column;
+  border-width: 1px;
+  border-style: solid;
+  border-color: white;
+  color: white;
+  font-size: 20px;
+}
+
+.nav_modal::-webkit-scrollbar {
+  display: none;
+}
+
+.nav_modal-header {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  width: 85.3211009174312%;
+  margin-left: 2rem;
+  margin-right: 2rem;
+}
+
+.nav_modal-title {
+  font-size: 20px;
+  align-self: center;
+  color: white;
+  text-align: left;
+}
+
+.nav_modal-body {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  height: inherit;
+}
+
+.nav_ul {
+  list-style-type: none;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  height: 50%;
+}
+
+.nav_li {
+  text-align: start;
+  font-size: 30px;
+}
+
+.divider {
+  height: 1px;
+  width: 95vw;
+  margin: 2px 0 8px 8px;
+  background-color: rgb(185, 185, 185);
+  padding: 0;
+}
+
+.modal_nav_link {
+  text-decoration: none;
+  padding: 2px 10px 2px 20px;
+  font-weight: bold;
+  color: white;
```

Comment:
I see that you give white color for links in the menu however, they are not white if you look at the Figma design, Please Kindly I suggest changing it to the color below as the Figma design
```CSS
color: #FFF5E1;
```
![image](https://user-images.githubusercontent.com/61790587/163661917-fc97c3bc-ed1b-4a51-83cf-42ebb448375f.png)